src/runtime/pkg/katautils/config.go

```diff
@@ -1059,7 +1059,6 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		BlockDeviceDriver:              blockDriver,
 		BlockDeviceCacheSet:            h.BlockDeviceCacheSet,
 		BlockDeviceCacheDirect:         h.BlockDeviceCacheDirect,
-		BlockDeviceCacheNoflush:        h.BlockDeviceCacheNoflush,
```

Comment:
Is this not supported for cloud-hypervisor? Also, the issue you mentioned on the `Fixes`  line #7823 talks about firecracker and not ccloud-hypervisor, so I was bit confused by that. In any case, will be useful to add this as a separate commit if the config needs to be removed explaining why it was removed.